rp_asos/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
```

Comment:
I think this would be a good opportunity to switch to Django's newer path object
```suggestion
from django.urls import path
```